src/optimizer.cpp

```diff
@@ -53,18 +54,40 @@ void Optimizer::getParams()
   getParam(vx_std_, "vx_std", 0.1);
   getParam(vy_std_, "vy_std", 0.1);
   getParam(wz_std_, "wz_std", 0.3);
+
+  getParam(controller_frequency_, "controller_frequency", 10.0);
```

Comment:
There should be no default set since it should always be valid at this point in the code. Also the parameter getter automatically sub-namespaces to `name.XYZ` https://github.com/artofnothingness/mppic/blob/850ec211d9f4effacd1a655ae17744131bc7656e/include/mppic/utils.hpp#L30 so this would be getting `plugin_name.controller_frequency` not `controller_frequency`

Reply:
Oh... sure..